tests/behat/fixtures/d7/docroot/sites/all/modules/custom/mysite_core/mysite_core.info

```diff
@@ -69,12 +70,17 @@ features[field_base][] = field_paragraph_link
 features[field_base][] = field_paragraph_media
 features[field_base][] = field_paragraph_title
 features[field_base][] = field_tags
+features[field_base][] = field_text_first
```

Comment:
please rename these fields to `field_fc_text_first` and `field_fc_text_second`. a simple search and replace should work

Reply:
ok, this actually requires more fields:

Please add `field_collection2` field to article
please add field `field_fc_text_third` and `field_fc_text_fourth` to `field_collection2` node field.

this is to the assert that the step actually allows to add values to multiple field collection fields at once:
```
	    And article content:
      | title           | field_collection:field_text_first;field_text_second | field_collection2:field_text_third;field_text_fourth |
      | Test collection | test_text1;test_text2                               |test_text3;test_text4                                |
```